crates/p2p/src/builder.rs

```diff
@@ -57,7 +57,8 @@ impl Builder {
             behaviour,
             local_peer_id,
             swarm::Config::with_tokio_executor()
-                .with_idle_connection_timeout(Duration::from_secs(3600 * 365)), // A YEAR
+                // .with_idle_connection_timeout(Duration::from_secs(60)),
+                .with_idle_connection_timeout(Duration::from_secs(3600 * 24 * 365)), // A YEAR
```

Comment:
BTW: I'm not sure having this infinite idle timeout is really a good idea. Why are we doing this?